packages/synapse-react-client/src/lib/containers/UserSearchBoxV2.tsx

```diff
@@ -25,23 +33,16 @@ export type UserSearchBoxProps = {
   filterPredicate?: (item: UserGroupHeader) => boolean
   placeholder?: string
   focusOnSelect?: boolean
+  clearOnSelection?: boolean
+  /* The principal ID of the current selected user or team. If null, no selection is made. If undefined, state will be handled internally */
+  value?: PropsValue<UserSearchBoxValueType> | undefined
```

Comment:
Caller can pass `null` as a value so the component doesn't store the most recently selected user. This gets us parity with `UserSearchBox` v1, so we now use it to pick accessors.

Reply:
Not a breaking change since the `undefined` behavior is unchanged